docs/source/plugin/bot.rst

```diff
@@ -96,6 +96,122 @@ the :meth:`~sopel.bot.SopelWrapper.action` method::
     bot.action('does something')
 
 
+Do it with style
+================
+
+.. Custom role definition to apply custom style to inline text
+
+.. role:: red
+    :class: red
+
+.. role:: strongred
+    :class: strong red
+
+.. role:: strike
+    :class: strike
+
+.. role:: strongem
+    :class: strong em
+
+.. role:: spoiler
+    :class: spoiler
+
+
+When the bot talks, replies, or acts, it can do it with style: colors,
+**bold**, *italic*, _underline_, :strike:`striked`, or ``monospace``. IRC
```

Comment:
Why don't the bold and italic examples (both here and below) use those custom rST roles/CSS classes? Why are the `.em` and `.strong` classes defined if not used by these roles? Is <tt>``</tt> always guaranteed to produce monospace text in every theme? Clearly, I have questions!

Reply:
I use them for doing both only, which is not possible with rst.

Reply:
And yes, double backquote is for inline monospace/code text. No sensible person would create a Sphinx theme without using a monospace font for that. And if they do, we should change that nonetheless.